### ISSUE TITLE
Unable to open dicomized data with wsidicom>=0.18.0

### ISSUE THREAD

**Issue by nickwww** [imi-bigpicture/wsidicom#155]
We have some dicomized testdata (used the wsidicomizer in early 2023 to dicomize CMU-1.svs) that can't be opened anymore using wsidicom>=0.18.0, showing the following error:

`AttributeError: 'Dataset' object has no attribute 'XOffsetInSlideCoordinateSystem'`

It seems that the old dicomized testdata has some missing data. Are there some mandatory dicom tags that need to be added to the data?

**Comment by erikogabrielsson** [comment]
Hi @nickwww,

I also noticed this problem, and made a fix for it. Unfortunately I missed to include the commit to the latest release. I will make a 0.19.1 as soon as possible with the fix. Please post any further issue you encounter that are similar to this. The missing attribute is mandatory for level image datasets, but we should allow it to be missing as it is not critical for most uses.

**Comment by erikogabrielsson** [comment]
Can you please provide me with the value of the `TotalPixelMatrixOriginSequence` in a file that failed?
```python
from pydicom import dcmread
ds = dcmread(file)
ds.TotalPixelMatrixOriginSequence[0]
```

**Comment by nickwww** [comment]
Thanks for the fast response, I've checked the`TotalPixelMatrixOriginSequence` for all dcm files in the folder and they all seem to be empty.

**Comment by erikogabrielsson** [comment]
Thanks. Now I have a working test and a fix for the bug. Will release as soon as the ci actions complete.

**Comment by erikogabrielsson** [comment]
Closed by #156
